copycat/runtime/src/main/java/org/apache/kafka/copycat/cli/CopycatStandalone.java

```diff
@@ -58,9 +60,9 @@ public static void main(String[] args) throws Exception {
         workerProps = !workerPropsFile.isEmpty() ? Utils.loadProps(workerPropsFile) : new Properties();
 
         WorkerConfig workerConfig = new WorkerConfig(workerProps);
-        Worker worker = new Worker(workerConfig);
+        Worker worker = new Worker(workerConfig, new FileOffsetBackingStore());
```

Comment:
Maybe a silly question, but any reason not to use KafkaOffsetBackingStore in any case?
We can always assume that Kafka is around, so why not use it in stand-alone case?